meteor/server/api/userActions.ts

```diff
@@ -559,6 +559,23 @@ class ServerUserActionAPI
 			}
 		)
 	}
+	async DEBUG_crashStudioWorker(userEvent: string, rundownPlaylistId: RundownPlaylistId) {
```

Comment:
What is this used for?

Reply:
Looks like it can be used to verify the workers are restarted after they crash.